src/Guild.js

```diff
@@ -25,6 +25,29 @@ class Guild {
     autoBind(this);
   }
 
+  getRoleNameByID (value) {
+    return Object.keys(this.roles).find(key => this.roles[key] === value);
+  }
+
+  createRole (id, name) {
+    this.roles[name] = id;
+  }
+
+  updateRoleName (id, name) {
+    const oldKey = this.getRoleNameByID(id);
+
+    if(this.isRole(name)) {
+      return;
+    }
+
+    this.roles[name] = this.roles[oldKey];
```

Comment:
You can just assign `id` here. While you do need the `oldKey` to delete in the next line, the result of `this.roles[oldKey]' will be the same as 'id'.

Reply:
Okay, i got it now.